pkg/schema/schema.go

```diff
@@ -40,7 +42,10 @@ type Obfuscate struct {
 	// static replacement where a detected mac address will be replaced by 'x'. Regex
 	// should be used with the 'regex' property that will define the regex, here the
 	// replacement also will be static by 'x'-ing out the matched string.
-	Type ObfuscateType `json:"type"`
+	Type ObfuscateType `json:"type" yaml:"type"`
+
+	// Report represents the replacement mapping of the obfuscated data
+	Report map[string]string `json:"report" yaml:"report"`
```

Comment:
yeah, that doesn't work :D This is generated code from the schema defined in json.

Reply:
we shouldn't call it `Report` either, maybe we can reuse the `ObfuscateReplacement` map for this? We can adjust the reporting to populate those fields.

Reply:
I would like to discuss on concluding the schema.json file based on the recent merges and accordingly generate this file.